src/model/signup.model.ts

```diff
@@ -13,6 +10,20 @@ export default {
     email: string,
     uid: string,
   ): Promise<User> {
+    if (username.length < 1) {
```

Comment:
These are all validations and would be done in the service file. 
@oddpharmacy is currently researching what kind of validations we need to add in issue #74 since something are already validated in the schema like; is user unique, the length,... if the username is not unique prisma will throw a PrismaError and will end up in the catch block.

So for error handling I would  add a try and catch block. You can check what best practice is for prisma models. But this logic would need to be implemented in the service file since these are validation checks.